Resources/Prototypes/Mood/drugs.yml

```diff
@@ -0,0 +1,34 @@
+# Drugs should be paired up in Trios of "Bonus, Penalty, Category"
+
+# Lotophagoi Oil
+- type: moodEffect
+  id: LotoTranscendence
+  moodChange: 30
+  timeout: 900 #15 minutes
+  moodletOnEnd: LotoEnthrallment
+  category: "LotophagoiAddiction"
+
+- type: moodEffect
+  id: LotoEnthrallment
+  moodChange: -30
+  timeout: 7200 #2 hours
+  category: "LotophagoiAddiction"
+
+- type: moodCategory
+  id: LotophagoiAddiction
+
+# Nicotine
+- type: moodEffect
+  id: NicotineBenefit
+  moodChange: 5
+  timeout: 600 #10 minutes
+  moodletOnEnd: NicotineWithdrawal
```

Comment:
So taking 1 Cig, mean oh boy... now your a permanant smoker for the rest of the round.

**GOOD LUCK**